src/ESPDash.h

```diff
@@ -125,15 +118,17 @@ class ESPDash{
 
     // Notify client side to update values
     void sendUpdates(bool force = false);
-
-    void refreshStatistics();
-
+    void refreshLayout() { sendUpdates(true); }
```

Comment:
Function definitions inside header file creates an issue when embedding ESP-DASH within a class. Moving definitions to cpp file is suggested.

Reply:
> embedding

Oh! I never experienced that... 
In fact, I am using that a lot (these are usually the kind of functions that could be inlined)
In which situation ? Thanks!